source/api_reference/single_payment_reference/index.html.md.erb

```diff
@@ -160,7 +160,7 @@ After you send your request to get information about a payment, you’ll receive
 | `total_amount`                                    | number          | Amount your user paid in pence, including corporate card fees.<br><br>`total_amount` only appears if you added a corporate card surcharge to the payment.<br><br>You can [read more about corporate card surcharges](/corporate_card_surcharges/#add-corporate-card-fees).                                                                   |
 | `fee`                                              | number          | The payment service provider's (PSP) transaction fee, in pence.<br><br>`fee` only appears when we have taken ('captured') the payment from the user or if their payment fails after they submitted their card details.<br><br>`fee` will not appear if your PSP is Worldpay or you are using an API key from a test service.<br><br>You can [read more about PSP fees](/reporting/#psp-fees).                                                                |
 | `net_amount`                                      | number          | The amount, in pence, that will be paid into your bank account after your payment service provider takes the `fee`.<br><br>`net_amount` can be a negative number if Stripe charges fees on a failed payment.<br><br>You can [read more about PSP fees](/reporting/#psp-fees). |
-| `provider_id`                                     | string          | The unique ID your payment service provider generated for this payment.<br><br>This is not the same as the `payment_id`.                                                                                                                                                                                                                         |
+| `provider_id`                                     | string          | The unique ID your payment service provider generated for this payment.<br><br>`provider_id` only appears after the user has submitted their payment details and GOV.UK Pay has tried to authorise the payment with your PSP.<br><br>This is not the same as the `payment_id`.                                                                                                                                                                                                                         |
```

Comment:
I think it might be a very accepted term in the context of the pay docs but "authorise" to me doesn't sound like plain english, I prefer allow but in this context it might not be right so feel free too ignore me, just a thought!

Reply:
Bit like reconciliation, 'authorise' is a specific (and not very plain English!) thing, so I don't think it can go. Curse these confusing words!